translations/harbour-editor-es.ts

```diff
@@ -267,15 +267,15 @@
     </message>
     <message>
         <source>Text auto-saved in:</source>
-        <translation type="unfinished"></translation>
+        <translation>Texto auto guardado en:</translation>
     </message>
     <message>
         <source>Quick note.</source>
-        <translation type="unfinished"></translation>
+        <translation>Nota rápida</translation>
     </message>
     <message>
         <source>~/Documents/harbour-editor-quickNote.txt</source>
-        <translation type="unfinished"></translation>
+        <translation>~/Documentos/harbour-editor-NotaRápida.txt</translation>
```

Comment:
Hm, my fault. This line does not need translation. I will delete it in next update.